jans-auth-server/server/src/main/java/io/jans/as/server/authorize/ws/rs/AuthorizeRestWebServiceImpl.java

```diff
@@ -784,19 +793,39 @@ private Response requestAuthorization(
         return builder.build();
     }
 
-    private ResponseMode extractResponseMode(String request) {
-        final Jwt jwt = Jwt.parseSilently(request);
-        if (jwt == null) {
-            return null;
+    @Nullable
+    private JsonWebResponse parseRequestToJwr(String request) {
+        if (request != null) {
```

Comment:
Lets use early exit. Means
```
if (request == null) {
    return null;
}
```
We should avoid nested ifs when possible. You can see already if-hell in other places.